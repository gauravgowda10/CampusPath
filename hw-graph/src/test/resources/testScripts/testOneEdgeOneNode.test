# Create a graph
CreateGraph oneEdgeOneNode

# Add a node
AddNode oneEdgeOneNode x

# Add an edge
AddEdge oneEdgeOneNode x x label

# Print the nodes in the graph and the outgoing edges from x
ListNodes oneEdgeOneNode
ListChildren oneEdgeOneNode x