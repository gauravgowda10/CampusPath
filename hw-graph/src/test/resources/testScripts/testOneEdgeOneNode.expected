# Create a graph
created graph oneEdgeOneNode

# Add a node
added node x to oneEdgeOneNode

# Add an edge
added edge label from x to x in oneEdgeOneNode

# Print the nodes in the graph and the outgoing edges from x
oneEdgeOneNode contains: x
the children of x in oneEdgeOneNode are: x(label)